client/src/containers/Dispositif/Dispositif.js

```diff
@@ -2035,29 +2035,36 @@ export class Dispositif extends Component {
                   {...this.state}
                 />
 
-                {this.state.disableEdit && !isMobile && (
+                {this.state.disableEdit && (
                   <>
                     {!printing && (
                       <FeedbackFooter
                         pushReaction={this.pushReaction}
                         didThank={didThank}
+                        thanks={
```

Comment:
si this.state.dispositif.merci est falsy tu veux que ce soir 0 et pas null non ? vu que cette variable représente le nombre de mercis